app/Models/User.php

```diff
@@ -352,7 +370,10 @@ public function isActiveMember()
                 ->where(function ($q) {
                     $q->whereNull('deleted_at')
                         ->orWhere('deleted_at', '>', date('Y-m-d H:i:s'));
-                })->get()
+                })
+                ->with('committee')
+                ->get()
+                ->where('committee.is_society', false)
```

Comment:
If I'm correct less queries are run if the society check is done within the with statement.

Reply:
See above.